features/support/branch_helpers.rb

```diff
@@ -21,8 +21,8 @@ def branches_for_repository repository
 
 
 # Creates and pushes a branch
-def create_branch branch_name, remote: true
-  run "git checkout -b #{branch_name} main"
+def create_branch branch_name, remote: true, commit: 'main'
```

Comment:
Use def with parentheses when there are parameters.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.